spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -134,6 +134,26 @@ describe("Router", function()
       assert.equal("api-1", res.headers["kong-api-name"])
     end)
 
+    it("routes by method-only if no Host header present", function()
+      -- a very limited HTTP client for sending requests without Host
+      -- based on the source of https://github.com/pintsized/lua-resty-http
+      local sock = ngx.socket.tcp()
+      finally(function() sock:close() end)
+      local ok, err = sock:connect(helpers.test_conf.proxy_ip,
```

Comment:
style: it is nice to see the effort to stay under 80 chars. As per CONTRIBUTING.md, the preferred way of doing so is to align subsequent arguments right under the first one :)